### ISSUE TITLE
config.json is missing uuid for exercise

### ISSUE THREAD

**Issue by iHiD** [exercism/swift#292]
Hello, 

As a result of #290 a new exercise was added to the config.json file. This broke the file due to a missing comma, which then broke the nexercism importer. I've just hot-fixed this missing comma as it's blocking us shipping nextercism, but I also noticed that the [new exercise](https://github.com/exercism/swift/blob/master/config.json#L545) is also missing a `uuid`, `core` and `unlocked_by` fields.

1) Could one of the @exercism/swift maintainers add these fields in for this exercise please :)
2) @kytrinyx - I'm guessing configlet isn't configured on this track? Or did it miss the comma? Could we rectify that either way please?
3) @ccare - Can we defend against this in the importer please? If broke during a full db seed, but I imagine it would be the same issue on an update too.

Thanks everyone 💙

**Comment by masters3d** [comment]
https://travis-ci.org/exercism/swift/jobs/265299577
@iHiD  The configlet did not fail even though it did report the config.json was not valid.  This should have failed.

Is there a way to check all the required fields are there in the JSON?

```
$ bin/fetch-configlet
The command "bin/fetch-configlet" exited with 0.
$ bin/configlet lint .
-> invalid config config.json -- invalid character '{' after array element
The command "bin/configlet lint ." exited with 0.
```

**Comment by masters3d** [comment]
How do I check if the UUID is unique across all exercises? https://github.com/exercism/docs/blob/39109e34d3590b4b9399600a57798601353e96d2/language-tracks/configuration/README.md

**Comment by ccare** [comment]
My understanding is that configlet should do that for you. I'll let @kytrinyx fill in the details when she's online. As @iHiD says, we'll also defend against it at our end (I thought we had, but obviously not).

**Comment by creaaa** [comment]
I'm Masa, who issued #290.
I am so sorry for this commit and the inconvenience this has been causing you.
To be honest, I was overlooking the process of configlet. I promise I never forget to do so anymore.
I can't apologize enough.

**Comment by masters3d** [comment]
@creaaa Hi Masa, This is not your fault. Please do not feel bad about it. This was a bug in the config checker that did not catch the new keys that were needed. The config checker should have failed the check when it saw that the config file was not valid. Thank you for your pull request and helping us find this bug. 👍

**Comment by iHiD** [comment]
@creaaa Don't worry at all :) When stuff like this breaks it's never any one person's fault and it's always the fault of systems not being good enough, not people making mistakes. This is why tests and CI and things like configlet exist. You can just take this as good learning about why those things are important, but don't feel bad about it! As @masters3d said, thank you for your contribution!

> How do I check if the UUID is unique across all exercises?
> Is there a way to check all the required fields are there in the JSON?

@masters3d Both of these will be configlet. Everything's under real active development atm so I imagine @kytrinyx is just playing catchup on things. I'm not sure what's written or not yet, but they will both eventually be the responsibility of it. In terms of uuid, if you generate a new one, it will be unique - the key is just not to copy and paste, which as you highlight, we need to check for.

Thanks for replying everyone - none of this is anything to worry about! :)

**Comment by kytrinyx** [comment]
Thanks so much for the contribution, and for finding all of these gaps in our CI process! No time like the present to get this all sorted out.

I'll make sure that configlet fails the build if necessary. It does some checking of UUIDs, I think (can't remember if we've implemented it or just opened the issue about it). I built a UUID checker service, but haven't had time to get Configlet to call it yet.
